packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/model/clientmodel/ClassType.java

```diff
@@ -515,6 +515,11 @@ private static ClassType.Builder getClassTypeBuilder(Class<?> classKey) {
         .name("ResponseInnerError")
         .jsonToken("JsonToken.START_OBJECT")
         .build();
+    public static final ClassType SYNC_POLLER_FACTORY
+        = new Builder().packageName("com.azure.core.management.polling").name("SyncPollerFactory").build();
+
+    public static final ClassType SYNC_POLLER
+        = new Builder().packageName("com.azure.core.util.polling").name("SyncPoller").build();
```

Comment:
nit, you should be able to use `knownClass` for the build?

Also better to move it near PollerFlux (if we had it here)

Reply:
Unfortunately, `http-client-generator-core` doesn't have `azure-core-management` as dependency... 

Previously we are doing: https://github.com/microsoft/typespec/blob/26a79df915832a5de7ab8c0ec2e63f92713475ed/packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/template/FluentServiceClientTemplate.java#L63

Reply:
Updated `SyncPoller` to use `knownClass`.